src/arch/host/CMakeLists.txt

```diff
@@ -6,3 +6,5 @@ target_include_directories(sof_public_headers INTERFACE ${PROJECT_SOURCE_DIR}/sr
 
 # C & ASM flags
 target_compile_options(sof_options INTERFACE -g -O3 -Wall -Werror -Wl,-EL -Wmissing-prototypes -Wimplicit-fallthrough=3 -Wpointer-arith)
+
+add_subdirectory(lib)
```

Comment:
Does this just make it work without including too much?

Reply:
@paulstelian97 This only includes 'lib' subdirectory in current tree = 'src/arch/host/lib', which only contains notifier.c and CMakeLists.txt, the notifier implementation for host. I've added these in this PR, but during all the rebasing this slipped somewhere.
It only adds what is really needed.